lib/cli.rb

```diff
@@ -179,13 +197,28 @@ def define_options(parser)
         parser.on('--package-with-playlist', TrueClass,
                   'Load playlist into HLKX project package',
                   &method(:package_with_playlist=))
+
+        parser.on('--session <path>', String,
+                  'Load session from workspace',
+                  &method(:session=))
+
+        parser.on('--latest-session', TrueClass,
+                  'Load previous session',
+                  &method(:latest_session=))
       end
       # rubocop:enable Metrics/AbcSize, Metrics/MethodLength
     end
 
     # class InstallOptions
-    class InstallOptions
-      attr_accessor :platform, :force, :skip_client, :drivers, :driver_path, :debug
+    class InstallOptions < T::Struct
+      include T::Sig
+
+      prop :platform, T.nilable(String)
+      prop :force, T::Boolean, default: false
+      prop :skip_client, T::Boolean, default: false
+      prop :drivers, T::Array[String], default: []
+      prop :driver_path, T.nilable(String)
+      prop :debug, T::Boolean, default: false
```

Comment:
Remove the default value assignments in `define_options` as you did for `CommonOptions`.